core-libs/setup/ssr/error-handlers/ssr-error-handler.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ErrorHandler, Injectable, inject } from '@angular/core';
+import { LoggerService } from '@spartacus/core';
+import { loggerEnabled } from '../logger';
+
+export const ssrErrorHandlerFactory = () => {
+  const isLoggerEnabled = inject(loggerEnabled);
+
+  return isLoggerEnabled ? new SsrErrorHandler() : new ErrorHandler();
+};
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SsrErrorHandler implements ErrorHandler {
```

Comment:
This class has nothing to do with SSR (but only to Logger which is a Core feature), can we move it to `@spartacus/core` ,e.g. to new folder `/error` ?
It can be e.g. `CxErrorHandler`